app/Database/seeds/settings.yml

```diff
@@ -305,3 +305,11 @@
   options: ''
   type: 'text'
   description: 'Discord public channel ID for broadcasat notifications'
+- key: pireps.advanced_fuel
+  name: 'Advanced Fuel Calculations'
+  group: pireps
+  value: false
+  options: ''
+  type: boolean
+  description: 'Enables remaining fuel amounts to be considered for fuel expenses'
+  
```

Comment:
Move this so it's below where the other pirep settings are. And make sure there is a newline at the end of the file

Reply:
Done